app/controllers/protocols_controller.rb

```diff
@@ -262,6 +262,18 @@ def clone
     end
     respond_to do |format|
       if !cloned.nil?
+
```

Comment:
We can move Creating Activity from controller to model to method `deep_clone_repository`